### PR TITLE
external site menu

### PR DESCRIPTION
**Pull request by Nesurion**

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -29,8 +31,71 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
     <script src="/js/jane.js"></script>
     <script src="/js/button.js"></script>
+
     <!-- Include all compiled plugins (below), or include individual files as needed -->
-    <!-- <script src="js/bootstrap.min.js"></script> -->
+    <script src="/js/bootstrap.min.js"></script>
+    <script src="/js/jasny-bootstrap.min.js"></script>
+    
+ <!--  <nav id="myNavmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation">
```
Comment (Nesurion) [inline views/index.erb]: remove comments plz

File: views/index.erb [hunk 2]
```diff
@@ -29,8 +31,71 @@
...
+      <ul class="nav navmenu-nav">
+        <li style="padding:10px"> Home Stuff </li>
+        <li><a href="#fritzbox">Frtiz!Box</a></li>
+      </ul>
+      <ul class="nav navmenu-nav">
+        <li style="padding:10px"> External Links </li>
+        <li><a href="#">Link</a></li>
+        <li><a href="#">Link</a></li>
+        <li><a href="#">Link</a></li>
+        <!-- <li class="dropdown">
```
Comment (Nesurion) [inline views/index.erb]: remove comments plz

File: views/index.erb [hunk 3]
```diff
@@ -29,8 +31,71 @@
...
+    </button>
+  </div> -->
+    <div id="sidebar" class="navmenu navmenu-default navmenu-fixed-left offcanvas">
+      <a class="navmenu-brand" href="#">External Links</a>
+      <ul class="nav navmenu-nav">
+        <li style="padding:10px"> Home Stuff </li>
+        <li><a href="#fritzbox">Frtiz!Box</a></li>
+      </ul>
+      <ul class="nav navmenu-nav">
+        <li style="padding:10px"> External Links </li>
```
Comment (Nesurion) [inline views/index.erb]: style directly in html is not cool. Could you write a css class for this and add it to `custom.css`?

**Comment by Lyr3x** [comment]
all done with 6daf65821e4ff6c4ad32948a16754fe86fb0c8c0

**Pull request merged**